clang/lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -2713,11 +2714,13 @@ llvm::Value *
 CGOpenMPRuntime::emitMessageClause(CodeGenFunction &CGF,
                                    const OMPMessageClause *MessageClause) {
   return emitMessageClause(
-      CGF, MessageClause ? MessageClause->getMessageString() : nullptr);
+      CGF, MessageClause ? MessageClause->getMessageString() : nullptr,
+      MessageClause->getBeginLoc());
```

Comment:
@ro-i you deferrence `MessageClause` after a nullptr check in the first time but unconditionally dereference right afterward. One of these has to be a bug.

Reply:
https://github.com/llvm/llvm-project/pull/164767